### DISCUSSION TITLE
Viewing on VLC

### DISCUSSION THREAD

**Discussion by BobC1954** [GerryDazoo/Slinger#242] [Q&A] [answered]
First, slinger working great!
But I am intrigued by something that happens with VLC (both Mac and Windoze versions).
I have a file "Slingboxes.m3u" which contains the streaming addresses for two of my boxes.
Nowhere in the file is the word "Slingbox", they are titled ProHD1 and HDS-600rs. 
This is initially displayed correctly when the playlist is opened, but changes to say "Slingbox" as soon as the stream starts.
So where does this title come from? It's nowhere in the m3u file, neither is it in config.ini
Does slingserver send this and then VLC changes to show, rather than the title saved in the m3u?

**Comment by GerryDazoo** [comment, marked as answer]
It's buried in the H.264 video stream. The first packet has some meta data in it and that's what the Slingbox is sending.

**Reply by BobC1954** [reply]
Ah! right, that explains it. I couldn't understand. I've been trying to get VLC to display the title as I had put in the m3u.

**Reply by GerryDazoo** [reply]
VLC has options up the ying yang. There might be something there to do what you want. :-)
